src/_base/helm/app/templates/application/webapp/service.yaml

```diff
@@ -13,3 +13,21 @@ spec:
     app.service: {{ .Values.resourcePrefix }}webapp
 status:
   loadBalancer: {}
+{{ if .Values.service.php_fpm_exporter }}
```

Comment:
Does helm like this two objects in one for v3? I had errors in v2 when doing this elsewhere

Reply:
I'm sure we already do it somewhere in OLP. Hmm. Trying to think where.

Reply:
No maybe not, I will split them just to be safe.